### ISSUE TITLE
2.2.3[BUG]

### ISSUE THREAD

**Issue by jamesmwlv** [bitfocus/companion#2080]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

On my 2019 MacBook Pro running Monterey 12.4, I just downloaded and installed the Mac Intel "Stable" 2.2.3 7/30/22. Once installed and launched, it fires up and show syncing, but never advances further then that.

### Steps To Reproduce

Launch browser from app button

### Expected Behavior

application to finish loading up

### Environment (please complete the following information)

```markdown
- OS:MOnterey 12.4
- Browser:Safari
- Companion Version:2.2.3
```

**Comment by Julusian** [comment]
What version were you on before?

Can you open the browser console https://support.happyfox.com/kb/article/882-accessing-the-browser-console-and-network-logs/, and send a screenshot of the log

**Comment by jamesmwlv** [comment]
Morning Julian,
I tried several version of Companion and had the issue on all, including the one working before upgrading to that last available version on the site. 

I then went and removed all companion files, folders and associations, reinstalled the latest and it is working. I believe I had a corrupted data file after the first update.

Thank you,
James M. Wadkins
702-275-0507 Cell
Sent from my iPhone
***@***.***
